components/automate-ui/src/app/modules/infra-proxy/policy-files/policy-files.component.html

```diff
@@ -2,7 +2,11 @@
   <chef-loading-spinner *ngIf="policyFilesListLoading" size="50"></chef-loading-spinner>
   <app-empty-state *ngIf="authFailure" moduleTitle="policyfiles" (resetKeyRedirection)="resetKeyTabRedirection($event)"></app-empty-state>
   <ng-container *ngIf="!policyFilesListLoading && !authFailure">
-    <chef-table>
+    <div class="empty-section" *ngIf="!policyFiles.length">
+      <img alt="No preview" src="/assets/img/no_preview.gif" />
+      <p>No policyFiles available</p>
```

Comment:
That capitalization seems awkward. I am not that familiar with the context but it might work better as plain English. (Do double check with @vinay033 or @vsingh-msys on this point, though.)
```suggestion
      <p>No policy files available</p>
```